### ISSUE TITLE
Limited thumb abduction

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11664]
# Limited thumb abduction

## Proposed definition

A reduced range of motion of thumb abduction (movement of the thumb away from the plane of the palm), as distinct from a fixed adduction contracture or a resting adducted-thumb position.

## Parent term

- `HP:0001376 Limitation of joint mobility`

## Synonyms

- Reduced thumb abduction

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617
- @evanc1648

**Comment by pnrobinson** [comment]
@evanc1648

Adding new term: 
- ID: HP:6001534
- label: Limited thumb abduction
- parents: [Limitation of joint mobility (HP:0001376)](https://hpo.jax.org/app/browse/term/HP:0001376)
- pmid: 
- def: A reduced range of motion of thumb abduction (movement of the thumb away from the plane of the palm), as distinct from a fixed adduction contracture or a resting adducted-thumb position.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11664
- synonyms: Reduced thumb abduction(NONE)
